package_test.go

```diff
@@ -52,6 +52,9 @@ func TestPackageBinfile(t *testing.T) {
 		}
 		got := pkg.Binfile()
 		want := filepath.Join(ctx.Bindir(), tt.want)
+		if pkg.gotargetos == "windows" {
```

Comment:
(Especially since `tt.goos` and `tt.goarch` aren't actually used in this block anywhere I can see. :smile:)